src/components/Events.tsx

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from "react";
+import type { Event } from "../model/Event";
+import style from "./Events.module.scss";
+import React from "react";
+
+export const Events: React.FC = () => {
+  const [events, setEvents] = useState<Event[]>([]);
+
+  useEffect(() => {
+    const apiUrl = "https://sportujspolu-api.onrender.com/api/v1/events";
+
+    fetch(apiUrl)
+      .then((response) => {
+        if (!response.ok) {
+          throw new Error("Network response was not ok");
+        }
+        return response.json();
+      })
+      .then((data: Event[]) => {
+        setEvents(data);
+      })
+      .catch((error) => {
+        console.error("Error:", error);
+      });
+  }, []);
+
```

Comment:
It would be great to show some loading state. Add it :). Doesnt have to be visually fancy.

Because right now it is quite empty when the data is being loaded
![Uploading CleanShot 2024-03-04 at 17.26.12.png…]()